File: Data Structures & Algorithms/three-integer-sum/submission-0.java
class Solution {
    public List<List<Integer>> threeSum(int[] nums) {
        Set<List<Integer>> arr = new HashSet<>();
        Arrays.sort(nums);
        for (int i = 0; i<nums.length; i++){
            for (int j = i + 1; j<nums.length; j++){
                for (int k = j + 1; k<nums.length; k++){
                    if (nums[i]+nums[j]+nums[k] == 0){
                        List<Integer> added = Arrays.asList(nums[i], nums[j], nums[k]);
                        arr.add(added);
                    }

                }
            }
        }
        return new ArrayList<>(arr); 
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
       count = {}
       for n in nums:
        count[n] = count.get(n, 0) + 1

       arr = []
       for n, cnt in count.items():
            arr.append([cnt, n])
       arr.sort()

       res = []
       while len(res) < k:
            res.append(arr.pop()[1])
       return res